README.md

```diff
@@ -18,11 +18,11 @@ A CLI for [Groonga](http://groonga.org/) using Readline. Supports Windows.
 
 For example:
 
-    $ grnlip groonga ~/.milkode/db/milkode.db
+    $ grnlip groonga %HOME%\.milkode\db\milkode.db
 
 If you use UTF-8 database on Windows, you should specify the following option to use multibyte characters:
 
-    > grnlip bin\groonga \Users\myokoym\.milkode\db\milkode.db utf-8
+    > grnlip bin\groonga C:\Users\myokoym\.milkode\db\milkode.db utf-8
```

Comment:
I think `C:` is needless because a path that start with `\` equals to the current partition's root such as `C:\`. Is there a reason to add `C:`?